Lib/textwrap.py

```diff
@@ -428,6 +427,7 @@ def cjk_wide(char):
     """Return True if char is Fullwidth or Wide, False otherwise.
     Fullwidth and Wide CJK chars are double-width.
     """
+    import unicodedata
```

Comment:
I think importing unicodedata, when wanted, for each char, is wrong.  If not imported at the top, it could be imported as a global in TextWrapper.__init__ when cjk is true.  (I am assuming that the convenience functions all instantiate TextWrapper.